webpack.config.dev.js

```diff
@@ -1,35 +1,36 @@
 // webpack.config.dev.js
 const ArcGISPlugin = require("@arcgis/webpack-plugin");
 const a = require("postcss-import");
-const b = require("tailwindcss");
+const b = require("tailwindcss")('./tailwind.config.js');
 const c = require("autoprefixer");
 
 module.exports = {
-    module: {
-      rules: [
-        {
-          test: /\.scss$/,
-          loader: "postcss-loader",
-          options: {
-            postcssOptions: {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loader: "postcss-loader",
+        options: {
+          postcssOptions: {
             ident: "postcss",
             syntax: "postcss-scss",
             plugins: [a, b, c],
-            }
-          },
-        },
-      ],
-    },
-    plugins: [
-        new ArcGISPlugin({
-          features: {
-            '3d': false
           }
-        })
-      ],
-      node: {
-        process: false,
-        global: false,
-        fs: "empty"
-      }
-  };
\ No newline at end of file
+        },
+      },
+    ],
+  },
+  plugins: [
+    new ArcGISPlugin({
+      features: {
+        '3d': false
+      },
+      locales: ['en']
```

Comment:
This does not make a difference. It still creates locale files for other languages.
Maybe because its deprecated from arcgis-webpack-plugin but still exists for dojo-webpack-plugin. Weird.